requirements.txt

```diff
@@ -1 +1,3 @@
 -e git+https://github.com/habnabit/txstatsd.git@master#egg=txStatsD
+ecdsa
+python-jose
```

Comment:
Should note that setup.py actually pegs the dep's, because I thought it might be nice to use ap-loadtester as a library.